material-components/material-components-android-examples
# [Reply] Adaptive Navigation Views

# Summary
- Added Adaptive Views for the Navigation Component
- Followed implementation with the Design Doc
- Implemented for Small (<600dp), Medium (600<->840) and Large (840+)

# Test
- Make sure everything is working when you 
- Pull Branch
- Run the Application

# Question
- Can we use Material Colors / Dynamic Colors for the Logo, even thought it is a PNG?
- Are we going to disable the Large Screen devices from closing the drawer?

# Screenshots

## Small
<img src="https://user-images.githubusercontent.com/32623283/141883037-ca1c7b2c-cf44-47a1-a499-2e1d86a9520c.png" width="300">

## Medium
<img src="https://user-images.githubusercontent.com/32623283/141883081-154ec669-a51c-4ac6-9ed4-a2b4f8504390.png" width="300">

## Large
<img src="https://user-images.githubusercontent.com/32623283/141883107-87fbcdb5-1fc5-4a39-89e7-9d3173ea9e38.png" width="300">
